lib/main.dart

```diff
@@ -9,20 +15,111 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
+      title: 'Playlister',
+      theme: darkTheme,
+      home: const MyHomePage(title: 'Playlister'),
+    );
+  }
+}
+
+class RecentlyUpdatedListItem extends StatelessWidget {
+  final String text;
+  final Color color;
+
+  const RecentlyUpdatedListItem({
+    Key? key,
+    required this.text,
+    required this.color,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 100,
+      margin: const EdgeInsets.symmetric(
+        horizontal: 10,
+      ),
+      child: ElevatedButton(
```

Comment:
This tile isnt semantically a button, but rather a surface that is clickable. Use `InkWell` instead